src/main/java/com/emc/object/s3/request/CopyObjectRequest.java

```diff
@@ -54,8 +54,6 @@
     private AccessControlList acl;
     private CannedAcl cannedAcl;
 
-    private String copyMode;
```

Comment:
Why is this change here?  Seems we are removing copyMode from CopyObjectRequest, but that should be in a separate branch/PR

Reply:
It seems to be introduced by merging PR#75 which didn't pickup copyMode change in master.